src/ess/polarization/he3.py

```diff
@@ -41,11 +49,10 @@ class He3CellTransmissionFraction(
     """Transmission fraction for a given cell"""
 
 
-# TODO rename and change to Up | Down | Unpolarized
-He3CellTransmissionFractionHasPolarizedIncomingBeam = NewType(
-    'He3CellTransmissionFractionHasPolarizedIncomingBeam', bool
-)
-"""Whether the transmission fraction is for a polarized incoming beam."""
+class He3AnalyzerTransmissionFraction(
+    sl.ScopeTwoParams[PolarizerSpin, PolarizationState, sc.DataArray], sc.DataArray
+):
+    """Transmission fraction of the analyzer with polarized incoming beam"""
```

Comment:
Hm, for me, the namings are unlogical. Do I see it correctly that you define `He3CellTransmissionFraction` as the transmission fraction for a 3He cell with an unpolarized incoming beam, vs. `He3AnalyzerTransmissionFraction` for an analyzer with polarized incoming beam?

Reply:
Yes. Which names would you suggest?